engines/scumm/scumm.cpp

```diff
@@ -2490,7 +2458,8 @@ void ScummEngine::scummLoop_handleSaveLoad() {
 
 			if (!_saveTemporaryState)
 				_lastSaveTime = _system->getMillis();
-		} else {
+		}
+		else {
```

Comment:
Why?

Reply:
The broken formatting? Sigh, seems to be some broken auto-formatting. I'll fix it...